backend/mlarchive/templates/archive/reports_subscribers.html

```diff
@@ -0,0 +1,73 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block title %}Mail Archive - Subscribers{% endblock %}
+{% block extrahead %}
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.21/css/jquery.dataTables.css">
```

Comment:
I think we should put some backpressure on having people's clients reaching to so many different places. Look to the datatracker for how it gathers and packages js and css.